src/pages/lessons/fundamentals/eth-intro-part-2.mdx

```diff
@@ -0,0 +1,279 @@
+---
+title: A Developer's Guide to Ethereum, Pt. 2
+description: Introduction to Ethereum with web3.py and Python.
+icons: ["ethereum"]
+---
+
+import { LessonHeader } from "../../../components/mdx/LessonHeader";
+import Layout from "../../../components/Layout";
+
+<Layout
+    title="A Developer's Guide to Ethereum, Pt. 2" 
+    description="Introduction to Ethereum with web3.py and Python." 
+>
+<LessonHeader title="A Developer's Guide to Ethereum, Pt. 2" />
+
+Welcome back! In the first installment of this series we covered a lot of ground
+while interacting with a simulated Ethereum network. At this point, you should
+have at least a fuzzy idea of how to answer these questions:
+
+- What's a blockchain and what’s in a block?
+- What makes Ethereum decentralized?
+- What is ether and why is it a necessary component of the network?
+
+In this lesson, we’ll build on these concepts and explore some of the implications
+for developers, so return to Part 1 if you skipped it or need a refresher.
+
+## What’s next? 
+
+We’re going to take a deeper look at how you can interact with the Ethereum network, starting with accounts. There are some significant differences between Ethereum accounts and Web 2.0 accounts.
+
+<Callout emoji="💡" size="md" variant="info">
+**Note**: “Web 2.0” was coined to describe the Internet era that introduced user-generated content, e.g., social media and blogs. Ethereum and other decentralized technologies are said to be a part of the next generation of the Internet: **Web 3.0**. The abbreviation, **Web3**, is used by libraries like web3.js and web3.py, and elsewhere around the ecosystem.
+</Callout>
+
+## Web2 vs. Web3
+
+It’s hard not to collect accounts in today’s web. You have one for each social media app, news site, delivery service, retailer, and transit provider — to name a few. Each of these accounts lives on the company's servers, which makes you subject to their terms and conditions, privacy policy, and security practices. Your account can be frozen, deleted, censored, or altered at the discretion of the host company.
+
+Web3 represents a paradigm shift for account management: you and you alone own your Ethereum accounts. When you create an account, it’s done outside of the scope of any company and can travel with you to multiple apps. In fact, creating an Ethereum account doesn’t require interaction with the Ethereum blockchain at all. Let’s create one now to prove it.
+
+<Callout emoji="💡" size="md" variant="info">
+**Note**: This exercise is purely for educational purposes. Don't store real value in an account unless you understand the security implications. Some mistakes cannot be undone! More context to come.
+</Callout>
+
+## Creating an account
+
+Same drill as last time: the concepts will be demonstrated in an IPython shell. If you’re not a Python developer, no problem. Just follow along conceptually.
+
+### Environment setup
+
+Three steps to set the stage:
+
```

Comment:
Marc, the word `eth-tester` is confusing. Do you mean `web3-tester`?

The sentence would also be much easier to understand,  by omitting the brackets (because it lowers the priority for the reader, or at least to me) and swap the clauses round. 'In case you didn't already in Part 1, install web3py, web3-tester and iPython:'

Reply:
I finally found what eth-tester was referring to...

Reply:
What I was saying here, Marc, is that I was completely lost here for a long time. It was very frustrating. Not good for learners. Let's come back to this at a later date - this is something we should document as it is fundamental to the review process.